extension.xml

```diff
@@ -4,22 +4,22 @@
 
 <root version='3.3' release='3'>
 	<properties>
-		<name>Clock Adjuster</name>
-		<version>2.4</version>
+		<name>Feature: Clock Adjuster</name>
+		<version>2.5</version>
 		<author>pr6i6est, bmos, and MeAndUnique</author>
 		<description>Provides an interface for easier time management</description>
 	</properties>
 	
-	<announcement text="Clock Adjuster:\rThis extension is a GM tool which provides an interface for easy time tracking and management.\nThis extension is a combined effort of PR6I6E6ST (Troy Monteith), bmos, and MeAndUnique" font="emotefont" />
+	<announcement text="https://www.fantasygrounds.com/forums/showthread.php?57561-Utility-Clock-Adjuster\rClock Adjuster v2.5:\rEvents, reminders, travel, and time.\rBy PR6I6E6ST, bmos, and MeAndUnique." font="emotefont" />
```

Comment:
Added a link to the forum page following convention I've seen in other extensions of yours.